resources/views/member/show.blade.php

```diff
@@ -55,7 +55,7 @@
                                     @include('partials/item', ['wowheadLink' => false])
                                     <small class="text-muted">
                                         on
-                                        <a href="{{route('character.show', ['guildSlug' => $guild->slug, 'name' => $character->name]) }}"
+                                        <a href="{{route('character.show', ['guildSlug' => $guild->slug, 'nameSlug' => $character->slug]) }}"

```

Comment:
Looks like an existing bug, but `$character->slug` needs to be changed to `$itemCharacter->slug`

Reply:
Updated